phpBB/includes/acp/acp_attachments.php

```diff
@@ -1228,7 +1228,7 @@ function main($id, $mode)
 						'FILETIME'			=> $user->format_date((int) $row['filetime']),
 						'REAL_FILENAME'		=> (!$row['in_message']) ? utf8_basename((string) $row['real_filename']) : '',
 						'PHYSICAL_FILENAME'	=> utf8_basename((string) $row['physical_filename']),
-						'EXT_GROUP_NAME'	=> (!empty($extensions[$row['extension']]['group_name'])) ? $user->lang['EXT_GROUP_' . $extensions[$row['extension']]['group_name']] : '',
+						'EXT_GROUP_NAME'   	=> (!empty($user->lang[$extensions[$row['extension']]['group_name']])) ? $user->lang['EXT_GROUP_' . $extensions[$row['extension']]['group_name']] : $extensions[$row['extension']]['group_name'],
```

Comment:
please use the lang function here (and not the array access)

Reply:
```suggestion
						'EXT_GROUP_NAME'	=> $this->language->is_set('EXT_GROUP_' . $extensions[$row['extension']]['group_name']) ?  $this->language->lang('EXT_GROUP_' . $extensions[$row['extension']]['group_name']) : $extensions[$row['extension']]['group_name'],
```

They are asking you to do something like this. (If you use this code you will have to add the language property in the top of the class and define it in the top of the main() method, like this:
`$this->language = $phpbb_container->get('language');`

Reply:
@phpbb-es

Reply:
@rubencm sorry, now I see this message ... I see that another ticket has been created and PR for you, this was commented by a Moderator of my forum some time ago.

Reply:
Yes, i forked your repository and fixed it, so your commit is still there and your contribution will be included in the repository, but if you still want to fix it here, i will close my pull request